Patch by Arne Fitzenreiter <arne_f@ipfire.org>
Fix PHYSDEVDRIVER from "Sundance IPG ..." to the modulname (ipg)

diff -Naur linux-3.2.33.org/drivers/net/ethernet/icplus/ipg.c linux-3.2.33/drivers/net/ethernet/icplus/ipg.c
--- linux-3.2.33.org/drivers/net/ethernet/icplus/ipg.c	2012-10-31 00:27:11.000000000 +0100
+++ linux-3.2.33/drivers/net/ethernet/icplus/ipg.c	2012-11-05 20:02:36.529101528 +0100
@@ -2304,7 +2304,7 @@
 }
 
 static struct pci_driver ipg_pci_driver = {
-	.name		= IPG_DRIVER_NAME,
+	.name		= DRV_NAME,
 	.id_table	= ipg_pci_tbl,
 	.probe		= ipg_probe,
 	.remove		= __devexit_p(ipg_remove),
